docs/integration.rst

```diff
@@ -169,6 +169,19 @@ AWS: Amazon Web Services
 Airflow has extensive support for Amazon Web Services. But note that the Hooks, Sensors and
 Operators are in the contrib section.
 
+AWS Athena
+''''''''
+
+- :ref:`AWSAthenaOperator` : Run Athena Query.
+
+.. _AWSAthenaOperator:
+
+AWSAthenaOperator
+"""""""""""""""""""
+
+.. autoclass:: airflow.contrib.operators.aws_athena_operator.AWSAthenaOperator
```

Comment:
Oh, EMR is out of place here. Could you just add a link to your classes (hook and operator) in `docs/code.rst` instead.

(Don't remove EMR from this section, just don't make any changes to this file)

Reply:
I'm moving Athena operator doc to the bottom and adding the same on `docs/code.rst`

Reply:
Please remove Athena from this section - EMR is out of place so we don't want to add more here.